edge-middleware/feature-flag-hypertune/app/api/route.ts

```diff
@@ -0,0 +1,17 @@
+import { NextResponse } from 'next/server'
```

Comment:
It might be worth moving this file to something like `/app/api/flags/route.ts` as it will otherwise be available as the `/api` endpoint which might be a bit confusing